src/Home.jsx

```diff
@@ -37,45 +30,28 @@ export default function Home(props) {
       result_events.data = result_events.data.slice(0, 20);
       setEvents(result_events.data);
     };
-    console.log("FETCHING FORUM DATA");
-    fetchPosts();
+
+    // remove these console.logs once we've verified that this gets called at all the right times. (wait till we have a client side routing directing here)
     console.log("FETCHING EVENTS DATA");
     fetchEvents();
   }, []);
 
   return (
     <div className={classes.root}>
-      <div className={classes.nav_post_container}>
```

Comment:
I just removed this div and moved its css to root because it seemed kind of redundant.